File: lib/models/my_task.dart
class MyTask {
  final int id;
  final String title;
  final DateTime createdAt;

  MyTask({required this.id, required this.title, required this.createdAt});

  factory MyTask.fromJson(Map<String, dynamic> json) {
    return MyTask(
      id: json['id'],
      title: json['title'],
      createdAt: DateTime.parse(json['created_at'] as String),
    );
  }
}

File: backend/app/models/my_task.py
from sqlalchemy import Column, Integer, String, DateTime, func

from ..core.database import Base


class MyTask(Base):
    __tablename__ = "my_tasks"

    id = Column(Integer, primary_key=True, autoincrement=True)
    user_id = Column(String, nullable=False, index=True)
    title = Column(String, nullable=False)
    created_at = Column(DateTime, nullable=False, server_default=func.now())

